scripts/csi_test/csi_tests_to_run

```diff
@@ -12,3 +12,5 @@ NodeUnstageVolume
 GetPluginCapabilities
 GetPluginInfo
 Probe
+ControllerPublishVolume
+Node
```

Comment:
what is "Node"?

Reply:
in this task there was a request to add 2 tests: ControllerPublishVolume and Node Service

Reply:
will it work if `Node` be `Node Service` instead?

Reply:
why Node be Node Service?

Reply:
this is the "tests to run" file, and I got the impression that these tests are the "Node Service" tests.
so if I understand correctly, the reason is to have the full name, which would be more clear.

Reply:
done

Reply:
@ArbelNathan since you opened this conversation, please resolve it once you feel it's resolved